libraries/AP_HAL_ChibiOS/hwdef/MatekH743/defaults.parm

```diff
@@ -0,0 +1,2 @@
+SERVO13_FUNCTION 120
+NTF_LED_TYPES 455
```

Comment:
Sampson, I think there may be an issue here: BOARD_PWM_COUNT_DEFAULT 13 is defined...it must be BOARD_PWM_COUNT_DEFAULT 12 for the default use of LED....I am not sure how to make an alternate config for the define ...maybe the one for pins works, if it does, then 
#define BOARD_PWM_COUNT_DEFAULT 12
#define BOARD_PWM_COUNT_DEFAULT 13 ALT(1)

might work... @Tridge would know for sure...

Reply:
This shouldn't be the issue, 
F765WING has these also

- BOARD_PWM_COUNT_DEFAULT 13
- SERVO13_FUNCTION 120
- NTF_LED_TYPES 455

Reply:
you are right.....this highlites a general issue however, setting up a servo output to a GPIO function appears in the SETUP/Servos screen (which needs to be increased beyond servo 8...I will add a Mission Planner issue) but does not mean that it actually will work unless the user changes BRD_PWM_COUNT also....will bring this up as an issue in devcall